include/DDPandoraPFANewProcessor.h

```diff
@@ -178,4 +181,11 @@ inline marlin::Processor *DDPandoraPFANewProcessor::newProcessor()
     return new DDPandoraPFANewProcessor;
 }
 
+//------------------------------------------------------------------------------------------------------------------------------------------
+//Added by T.Pasquier
+/**
+*  @brief  Register the LCContent plugins outside their original function. Temporary fix. Would be better to change the plugins registration directly in LCContent
+*/
+pandora::StatusCode PandoraHack(const pandora::Pandora &pandora);
```

Comment:
```suggestion
pandora::StatusCode AprilLCContentPluginRegistration(const pandora::Pandora &pandora);
```
(or a more fitting name)

Would at least hide a bit that this is a "hack", while also making it rather clear from the function name what is happening (on a high level).

Reply:
I updated the name in the commit "Fixed the factory issue" (d793e70)